domain/model/service/service.go

```diff
@@ -44,6 +44,8 @@ func NewService(st State) *Service {
 // - modelerrors.AlreadyExists: When the model uuid is already in use or a model
 // with the same name and owner already exists.
 // - errors.NotFound: When the cloud, cloud region, or credential do not exist.
+// - [github.com/juju/juju/domain/user/errors.NotFound] when the owner of the
```

Comment:
Shouldn't this be `[domain.user.errors.NotFound]`

Reply:
No see here https://pkg.go.dev/github.com/fluhus/godoc-tricks#Links.

Needs to be full import path.

Reply:
I could bring it in as an import into the package but that isn't perfect either.